cooperator/data/mail_template_data.xml

```diff
@@ -271,7 +271,7 @@
             ><![CDATA[
 <div style="font-family: 'Lucica Grande', Ubuntu, Arial, Verdana, sans-serif; font-size: 12px; color: rgb(34, 34, 34); background-color: #FFF; ">
 
-    <p>Hello ${object.firstname},</p>
+    <p>Hello ${object.subscription_request_ids[-1].firstname},</p>
```

Comment:
Why the [-1] ? The order of moves is `_order = 'date desc, name desc, id desc'` so you'll end up with the oldest request, is that what you want ? 
Also, why do you index here and not on line 32 ?

Reply:
good catch, thanks! fixed.

on line 32, it is the `account.move` model, not the `res.partner` model, so the subscription request is directly accessible.

Reply:
Ok sorry, pas vu que c'était deux templates différents.